dev-libs/libratbag/libratbag-0.13.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit meson python-single-r1 udev
+
+DESCRIPTION="Library to configure gaming mice"
+HOMEPAGE="https://github.com/libratbag/libratbag"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc elogind systemd test"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	^^ ( systemd elogind )
+"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
```

Comment:
Python is used at build time, so `${PYTHON_DEPS}` is needed in `BDEPEND`.
https://github.com/libratbag/libratbag/blob/1097b1ba686e6a0c62868737bc767b34a039b110/meson.build#L541-L547
https://github.com/libratbag/libratbag/blob/1097b1ba686e6a0c62868737bc767b34a039b110/meson.build#L562-L564
https://github.com/libratbag/libratbag/blob/1097b1ba686e6a0c62868737bc767b34a039b110/meson.build#L667-L669

Reply:
```
tools/ratbagd.py:from gi.repository import Gio, GLib, GObject
tools/toolbox.py:from gi.repository import GLib
tools/toolbox.py:    from gi.repository import Gio
```

`tools/toolbox.py` is used to create `toolbox.py`, which seems to be used only at build time.

`tools/merge_ratbagd.py` seems to create `ratbagctl` (installed) from `'tools/ratbagctl.in.in` and `tools/ratbagd.py`.
So `from gi.repository import Gio, GLib, GObject` is present in installed file.

Assuming that above analysis is correct, both `BDEPEND` and `RDEPEND` need additional dependencies:
- For general `gi` module infrastructure, `dev-python/pygobject:3` is needed (as `$(python_gen_cond_dep 'dev-python/pygobject:3[${PYTHON_MULTI_USEDEP}]')`).
- For importing modules specified after `from gi.repository import` (in this case `Gio`, `GLib`, `GObject`), appropriate `/usr/share/gir-1.0/*.gir` files are needed. `dev-libs/gobject-introspection` provides `/usr/share/gir-1.0/Gio-2.0.gir`, `/usr/share/gir-1.0/GObject-2.0.gir`, `/usr/share/gir-1.0/GLib-2.0.gir`.

Reply:
Fixed for both ebuilds. 

- Python is also required by piper at build time.